src/hooks/useRecordingSequence.ts

```diff
@@ -97,6 +102,19 @@ export default function useRecordingSequence(
         return
       }
 
+      // We want to stop the recording when the left mouse button is pressed. Currently, always stops the recording
+      if (enumVal === MouseButton.Left) {
+        toast({
+          title: `Sequence recording stopped`,
+          description: `To record Mouse Button 1, insert the button from the left panel.`,
+          status: 'info',
+          duration: 4000,
+          isClosable: true
+        })
+        setRecording(false)
+        return
+      }
+
```

Comment:
add a notification and stop the recording of the sequence if you press the LMB. This should make it a lot nicer (ft. Pola)